lib/problems.rb

```diff
@@ -2,8 +2,32 @@
 
 # Time Complexity: ?
 # Space Complexity: ?
+
+# unfinished not passing, but can you give some feedback on my approach
+
 def balanced(string)
```

Comment:
Maybe set up a dictionary where the keys are the close markers `), }, ]` and the values are the matching open characters.  Then you can do if the current character isn't a key in the hash you can push it onto the stack, if it IS a close character, you can then compare 

```ruby
return false if char_hash[char]  != stack.pop()
```